server/api/middleware.js

```diff
@@ -0,0 +1,36 @@
+const adminMiddleware = (req, res, next) => {
+  if (!req.session.user.isAdmin) {
+    const err = new Error('Forbidden, request requires elevated permissions')
+    err.status = 401
+
+    return next(err)
+  }
+  next()
+}
+
+const userCheckMiddleware = (req, res, next) => {
```

Comment:
I think this could be named more specifically. maybe like `isSelf` or `userIsSelf` or something

Reply:
renamed appropriately